content/influxdb/v2.0/reference/flux/stdlib/profiler/_index.md

```diff
@@ -21,6 +21,9 @@ Import the `profiler` package:
 import "profiler"
 ```
 
+Each enabled profiler appends a table to the output stream of tables containing
+data returned by the profiler.
+
 ## Options
 The Profiler package includes the following options:
```

Comment:
Looks like there's no list of options after this statement (other than the `enabledProfilers` header)

Reply:
There is currently only one option for this package, but that could change in the future. The heading follows the pattern established in other packages.